### ISSUE TITLE
[V2] userIsInRole should check that id is a string

### ISSUE THREAD

**Issue by Floriferous** [Meteor-Community-Packages/meteor-roles#323]
The following code doesn't work as I would expect:

```js
const userId = Meteor.users.insert({});
Roles.setUserRoles(userId, 'admin');

Roles.userIsInRole(() => {}, 'admin'); // Returns true
```

The line [here](https://github.com/Meteor-Community-Packages/meteor-roles/blob/ce66e2488b45ebd894b2ea9dccd73fd776807ad5/roles/roles_common.js#L821) should check that the id is a string.

This used to fail properly in Roles V1.

**Comment by SimonSimCity** [comment]
Is this only related to v2 or also to v3?

**Comment by Floriferous** [comment]
Not sure, haven’t looked at v3 code.

**Comment by SimonSimCity** [comment]
Fixed in #324.
